src/tests.ts

```diff
@@ -107,3 +107,4 @@ import './types_test';
 import './util_test';
 import './variable_test';
 import './version_test';
+import './worker/worker_test';
```

Comment:
This error may be caused by the invalid import path here?
```
Step #1 - "build": Error: Looks like you added a new test file. Please run `yarn build` to regenerate src/tests.ts
```

```suggestion
import './worker_test';
```

Reply:
Great! Thanks for pointing out that!